_posts/en/newsletters/2020-09-23-newsletter.md

```diff
@@ -0,0 +1,291 @@
+---
+title: 'Bitcoin Optech Newsletter #116'
+permalink: /en/newsletters/2020/09/23/
+name: 2020-09-23-newsletter
+slug: 2020-09-23-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes a proposed soft fork to enable a new
+type of fee bumping and summarizes research into scripts that can never
+be spent because they require satisfying both timelocks and heightlocks.
+Also included are our regular sections with summaries of updates to
+services and client software, a list of new releases and release
+candidates, and changes to popular Bitcoin infrastructure software.
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **Transaction fee sponsorship:** Jeremy Rubin [posted][rubin sponsor
+  bumping] a proposal to the Bitcoin-Dev mailing list for a soft fork to
+  add a new fee bumping mechanism that could be less vulnerable to abuse
+  than the existing Replace-by-Fee ([RBF][topic rbf]) and
+  Child-Pays-For-Parent ([CPFP][topic cpfp]) methods.  Potential abuses
+  against the existing systems, such as the use of [transaction
+  pinning][topic transaction pinning], can be used to attack contract
+  protocols.  This has led to partial solutions such as [anchor
+  outputs][topic anchor outputs] that work with specific protocols (e.g.
+  two-party LN channels) but aren't easy to generalize and so are
+  somewhat unsatisfactory.
+
+    At the consensus layer, Rubin proposes allowing transactions to
+    optionally contain a special final output that commits to the txids
+    of one or more other unconfirmed transactions.  The transaction with
+    the special output, a *sponsor* transaction, may only be included in
+    a valid block if every one of the transactions it *sponsored* is
+    also included in that same block.  This means a miner who wants the
+    high feerate from a sponsor transaction will be incentivized to also
+    confirm the sponsored transactions even if those transactions pay a
+    low feerate.  Beyond their special sponsorship output, sponsor
+    transactions are normal Bitcoin transactions.
+
+    At the policy layer that controls mempool acceptance and transaction
```

Comment:
It's not obvious that 'that controls mempool ....' is the definition of 'policy layer' and not a qualifier (i.e. the reader may ask "is there a different policy layer that doesn't control mempool ...). Can this be rephrased to make it obvious that it's a definition, e.g. "At the policy layer (the rules which control mempool...), Rubin suggests..."